src/components/Search/Search.js

```diff
@@ -1,103 +1,69 @@
-import React, { Component, Fragment } from 'react';
-import { Row, Col } from 'antd';
+import React, { Component } from 'react';
+import { Layout } from 'antd';
 import SearchBar from './SearchBar';
+import FileUploader from './FileUploader';
+import UploadSwitch from './UploadSwitch';
 import { styles } from '../../utils/constants';
 
 class Search extends Component {
     constructor(props) {
         super(props);
         this.state = {
             fileName: '',
+            showFileUpload: false,
         }
     }
 
-    handleCountySelect = (i) => {
-        this.props.onCountySelect(i);
+    handleCountySelect = (event) => {
+        this.props.onCountySelect(event);
     }
 
-    handleUpload = (event) => {
-        if (event.target.files.length > 0) {
-            const file = event.target.files[0]
-            if (this.validateSize(file) && this.validateType(file)) {
-                const fileReader = new FileReader();
-                fileReader.onload = () => {
-                    const json = JSON.parse(fileReader.result);
-                    this.props.onFileUpload(json);
-                    console.log('json', json)
-                    this.setState({'fileName': file.name})
-                }
-                fileReader.readAsText(file)
-            }
-        }
+    handleUpload = (event, name) => {
+        console.log('handling Upload in Search', event, name)
+        this.props.onFileUpload(event, name);
     }
 
-    validateSize = (file) => {
-        let size = 100000000;  //100mb
-        let error = '';
-        if (file.size > size) {
-            error = 'File is too large, please upload a smaller file';
-            // react-toastify beautifies notifications
-            alert(error);
-        } else {
-            return true;
-        }
-    }
-
-    validateType = (file) => {
-        let error = '';
-        if (file.type !== 'application/json') {
-            error = 'Please upload json file';
-            // react-toastify beautifies notifications
-            alert(error);
-        } else {
-            return true;
-        }
+    handleUploadToggle = () => {
```

Comment:
`this.state` can be accessed here as well

An alternate way to write this
`handleUploadToggle = () => this.setState({showFileUpload: !this.state.showFileUpload});`